packages/web-api/src/response/AdminAnalyticsGetFileResponse.ts

```diff
@@ -24,6 +16,7 @@ export interface ResponseMetadata {
 
 export interface AdminAnalyticsMemberDetails {
   enterprise_id:                 string;
+  team_id:                       string;
```

Comment:
Awesome, I think I will spend the required time to review all the java-slack-sdk responses, make sure the node sdk has parity with it and document them as well as I go.

Reply:
I will spend some time setting up the full java-slack-sdk integration tests as I want to see if it would be possible to somehow generate documentation for these fields with a semi-automated method as well.

@seratch the documentation in the generated Java fields, like [here](https://github.com/slackapi/java-slack-sdk/blob/main/slack-api-client/src/main/java/com/slack/api/methods/response/admin/analytics/AdminAnalyticsGetFileResponse.java#L144-L148), are they hand-written?

Reply:
@filmaj Yes, they are. The benefits of the tests in the SDK is to detect unknown properties. When enabling [this detection mode](https://github.com/slackapi/java-slack-sdk/blob/v1.34.0/slack-api-client/src/test/java/config/SlackTestConfig.java#L81C16-L81C42), JSON parser throws an exeption with a newly added property information. The unit test results in a failure, and then we, developers, can modfiy the code until it passes the test without any exceptions.

Reply:
I see, that makes sense! Maintainers of this project should follow the same approach. I will do my best to document the fields in a nice way as I work through this.